php/Terminus/Workflow.php

```diff
@@ -123,4 +122,25 @@ public function status($key=null) {
      return $this->status;
    }
 
+   public function isFinished() {
+     if ($this->status->result) {
```

Comment:
Oh, but I wanted to coerce it to true/false. I can do `return !!($this->status->result);